src/scripts/clipperUI/componentBase.ts

```diff
@@ -105,20 +105,61 @@ export abstract class ComponentBase<TState, TProps> {
 				} else if (e.which === Constants.KeyCodes.tab) {
 					// Since they are using the keyboard, revert to the default value of the outline so it is visible
 					element.style.outlineStyle = "";
+				} else if (e.which === Constants.KeyCodes.up) {
+					if ( setNameForArrowKeyNav === undefined) {
```

Comment:
We should avoid repeated code as much as possible. You can do that by adding the check before checking if e.which === up/down. Also, you should check for null and empty string cases, which can be done with `if (!setNameForArrowKeyNav)

Reply:
Also please remove whitespace after `if ( `